packages/quick_actions/quick_actions_ios/example/ios/RunnerTests/Fixtures.m

```diff
@@ -0,0 +1,45 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "Fixtures.h"
```

Comment:
To clarify, there are 3 options here: 
1. put these data under a separate fixtures file (to be reused by all)
2. put these data as [helpers in each test file](https://abseil.io/tips/122) (to be reused by tests within the test file)
3. put these data inside each test function (no reuse)

Are you referring to option 2 or 3? 

I don't have a strong preference here (art, not science).

Reply:
^ feel free to chime in @cyanglaz and @stuartmorgan. Let's decide on a convention and use it onwards.

Reply:
I was referring to 2 but if I were writing these myself I would do 3. Art not science, as you say.  I don't have a preference, I just think they should be in the same file to avoid
> “OK, this is a FrobberTest, where’s that defined … oh, this file. Great.”

but when expectations fail to be able to easily see what the actual vs expectation is.

Reply:
Personal opinion. What I usually do is asking myself: "Does having the result inline break the readability of the test? ". For example, if the result of the test is a huge json (e.g. 100+lines) and there are a lot of these jsons, i would put the jsons in a different file; if the result is about 5 lines, i would put them inline or in the same file.

Reply:
I will put them inside each function then